backend/fichier/api.go

```diff
@@ -90,6 +90,7 @@ func (f *Fs) getDownloadToken(ctx context.Context, url string) (*GetTokenRespons
 	request := DownloadRequest{
 		URL:    url,
 		Single: 1,
+		Pass: f.opt.SharedPassword,
```

Comment:
This will be passing empty string when SharedPassword is not set - is that OK?

Reply:
I didn't have any problems with that.
However I made another change and added `omitempty`.
I have not tried to set a password using the API but in the web UI an empty string removes the password.